test/gtest/ucp/test_ucp_atomic.cc

```diff
@@ -124,17 +126,26 @@ class test_ucp_atomic : public test_ucp_memheap {
 
     virtual void init() {
         int atomic_mode = get_variant_value() & ATOMIC_MODE;
+        int odp         = get_variant_value() & ODP;
         const char *atomic_mode_cfg =
                         (atomic_mode == UCP_ATOMIC_MODE_CPU)    ? "cpu" :
                         (atomic_mode == UCP_ATOMIC_MODE_DEVICE) ? "device" :
                         (atomic_mode == UCP_ATOMIC_MODE_GUESS)  ? "guess" :
                         "";
         modify_config("ATOMIC_MODE", atomic_mode_cfg);
+
+        if (odp) {
+            modify_config("REG_NONBLOCK_MEM_TYPES", "host,cuda-managed");
+            modify_config("IB_ODP_MEM_TYPES", "host,cuda-managed",
```

Comment:
afair, we can't set odp for cuda-managed on non-grace system

Reply:
Thanks, removed.